Documentation/En/Index.xyl

```diff
@@ -140,8 +140,8 @@
   <h3 id="Test_framework" for="main-toc">Test framework</h3>
 
   <p>Hoa did not developed its own test framework for several reasons. The main
-  one is this is a laboreous task. It implies more development, more tools to
-  maintain, less people focused on Hoa etc. Instead, we prefer to choose a
+  one is that this is a laborious task. It implies more development, more tools
```

Comment:
“that” is not necessary. “laboreous” is correct but this is not the most common form in the wild, so agree!